bokeh/util/compiling.py

```diff
@@ -0,0 +1,274 @@
+from __future__ import absolute_import
```

Comment:
I have a possibly odd but definitely strong aversion to having module names be gerunds (i.e "xyz-*ing*") What about `util.compile` or `util.compiler` or `util.jsbuild` ?